lib/Agrammon/Web/Service.pm6

```diff
@@ -350,6 +350,7 @@ class Agrammon::Web::Service {
         my $results = self!get-outputs($user, $dataset-name)<results>;
         my $validation-errors = self!get-outputs($user, $dataset-name)<validation-errors>;
         warn '**** Got ' ~  $validation-errors.elems ~ ' input validation errors' if $validation-errors;
+        dd 'Validation errors:', $validation-errors if $validation-errors;
```

Comment:
Debug leftover?

Reply:
No, on purpose for the moment. I think it is useful to have those in the log file on the server (see also comment on this PR at the top).
Added a comment.